windows/src/desktop/kmshell/xml/config.js

```diff
@@ -194,12 +210,37 @@ document.addEventListener("DOMContentLoaded", windowResize);
 
   function list_detail(event,n) {
     var k = document.getElementById('list_'+n);
-    $(k).toggleClass('expanded');
+    list_expanded_selected (k);
     k.focus();
     save_state();
     return false;
   }
 
+  function list_expanded_selected (n){
+    if(loading_state) return;
+    var keyboards = $('div.list_item');
+    keyboards.each(function(index) {
+      if(this.id == n.id) {
+        $(this).toggleClass('expanded');
```

Comment:
```suggestion
        $(this).addClass('expanded');
```

Is that safer than a toggle?

Reply:
The desired behaviour here is to toggle the class for the matching id (not add) and remove it for all other ids. At most one keyboard item is to be expanded at a time.